src/scroll-animate-directive.js~

```diff
@@ -0,0 +1,124 @@
+'use strict';
```

Comment:
This file shouldn't be in the commit. I believe this is a backup created automatically by your text editor.